docs/csharp/misc/cs1570.md

```diff
@@ -2,43 +2,35 @@
 description: "Compiler Warning (level 1) CS1570"
 title: "Compiler Warning (level 1) CS1570"
 ms.date: 07/20/2015
-f1_keywords: 
+f1_keywords:
   - "CS1570"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS1570"
-ms.assetid: a121d5c4-8b90-4cda-af5b-6ba8f23b2b1e
 ---
 # Compiler Warning (level 1) CS1570
 
-XML comment on 'construct' has badly formed XML — 'reason'  
-  
- When using [**DocumentationFile**](../language-reference/compiler-options/output.md#documentationfile), any comments in the source code must be in XML. Any error with your XML markup will generate CS1570. For example:  
-  
-- If you are passing a string to a **cref**, such as in an [\<exception>](../programming-guide/xmldoc/exception.md) tag, the string must be enclosed in double quotation marks.  
-  
-- If you are using a tag, such as [\<seealso>](../programming-guide/xmldoc/seealso.md), which does not have a closing tag, you must specify a forward slash before the closing angle bracket.  
-  
-- If you need to use a greater-than or less-than symbol in the text of description, you need to represent them with **&gt;** or **&lt;**.  
-  
-- The file or path attribute on an [\<include>](../programming-guide/xmldoc/include.md) tag was missing or improperly formed.  
-  
- The following sample generates CS1570:  
-  
-```csharp  
-// CS1570.cs  
-// compile with: /W:1  
-namespace ns  
-{  
-   // the following line generates CS1570  
-   /// <summary> returns true if < 5 </summary>  
-   // try this instead  
-   // /// <summary> returns true if <5 </summary>  
-  
-   public class MyClass  
-   {  
-      public static void Main ()  
-      {  
-      }  
-   }  
-}  
+XML comment on 'construct' has badly formed XML — 'reason'
+
+When using [**DocumentationFile**](../language-reference/compiler-options/output.md#documentationfile), any comments in the source code must be in XML. Any error with your XML markup will generate CS1570. For example:
+
+- If you are passing a string to a **cref**, such as in an [\<exception>](../programming-guide/xmldoc/exception.md) tag, the string must be enclosed in double quotation marks.
+
+- If you are using a tag, such as [\<seealso>](../programming-guide/xmldoc/seealso.md), which does not have a closing tag, you must specify a forward slash before the closing angle bracket.
+
+- If you need to use a greater-than or less-than symbol in the text of description, you need to represent them with `&gt;` or `&lt;`. Alternatively, you can use CDATA.
```

Comment:
I mentioned CDATA here as an alternative. It's a good option, especially if there are lots of special symbols that needs to be escaped (commonly used with `<code>`)